### DISCUSSION TITLE
Images created via svg have defects

### DISCUSSION THREAD

**Discussion by doudouodong** [libvips/libvips#3669] [General]
## Questions, enhancements, tips, etc.

https://github.com/libvips/libvips/assets/47808515/f57d2b1c-d545-4341-92aa-0d514587c025

```C++
void createSvg()
{
    std::string svgData = "<svg viewBox=\"0 0 1510 730\"><rect x=\"15\" y=\"15\" width=\"1480\" height=\"700\" rx=\"50\" ry=\"50\" style=\"stroke:#9013FEFF; fill:#FB7701FF; stroke-width:30px\"/></svg>";
    vips::VSource source = vips::VSource::new_from_memory(svgData.c_str(), svgData.length());
    vips::VImage renderImage = vips::VImage::svgload_source(source);
    renderImage.write_to_file("xxx.png");
}
```

**Screenshots**
https://github.com/libvips/libvips/assets/47808515/f57d2b1c-d545-4341-92aa-0d514587c025

**Environment**
(please complete the following information)
 - OS: [Ubuntu 16.04, MacOS 10.13]
 - Vips: [e.g. 8.14.2 8.14.3, etc]

**Additional context**

**Comment by lovell** [comment]
It looks like whatever software is rendering the transparent background of the PNG is using a checked pattern to do so. This is probably unrelated to libvips (or librsvg).

**Reply by doudouodong** [reply]
When I render svg with chrome, I don't have this problem。

<!DOCTYPE html>
<html>
<body>

<h1>My first SVG</h1>

<svg viewBox="0 0 1510 730"><rect x="15" y="15" width="1480" height="700" rx="50" ry="50" style="stroke:#9013FEFF; fill:#FB7701FF; stroke-width:30px"/></svg>
 
</body>
</html>

**Reply by jcupitt** [reply]
@lovell is right, libvips is not rendering that checkerboard background, it's coming from your image viewer. Try a different viewer.

For example, with your test SVG:

```svg
<svg viewBox="0 0 1510 730">
  <rect x="15" y="15" 
	width="1480" height="700" 
        rx="50" ry="50" 
	style="stroke:#9013FEFF; fill:#FB7701FF; stroke-width:30px"
  />
</svg>
```

I can run:

```
$ vips copy x.svg x.png
```

To make:

![x](https://github.com/libvips/libvips/assets/580843/22f1c9a1-9398-4c78-b68e-33ffabf95bef)

No checkerboard pattern.
